From 000912d45613e278d7279bc3624aff906fd8d91b Mon Sep 17 00:00:00 2001
From: Stefan Saraev <stefan@saraev.ca>
Date: Wed, 11 Mar 2015 20:56:15 +0200
Subject: [PATCH 06/12] dont set _NET_WM_STATE_FULLSCREEN

I dont remember why I added this. I guess it was for
some 3rdparty stuff that needed to go fullscreen but
xbmc was acting weird..

this should probably be removed after OE 6
---
 xbmc/windowing/X11/WinSystemX11.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/xbmc/windowing/X11/WinSystemX11.cpp b/xbmc/windowing/X11/WinSystemX11.cpp
index 05f36ee..1714422 100644
--- a/xbmc/windowing/X11/WinSystemX11.cpp
+++ b/xbmc/windowing/X11/WinSystemX11.cpp
@@ -1155,8 +1155,6 @@ bool CWinSystemX11::SetWindow(int width, int height, bool fullscreen, const std:
 
     if (fullscreen && hasWM)
     {
-      Atom fs = XInternAtom(m_dpy, "_NET_WM_STATE_FULLSCREEN", True);
-      XChangeProperty(m_dpy, m_mainWindow, XInternAtom(m_dpy, "_NET_WM_STATE", True), XA_ATOM, 32, PropModeReplace, (unsigned char *) &fs, 1);
       // disable desktop compositing for KDE, when Kodi is in full-screen mode
       int one = 1;
       XChangeProperty(m_dpy, m_mainWindow, XInternAtom(m_dpy, "_KDE_NET_WM_BLOCK_COMPOSITING", True), XA_CARDINAL, 32,
-- 
2.1.4

